packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -143,27 +149,20 @@ export class AliasGatherer implements ParametersGatherer<Alias> {
       prompt: nls.localize(
         'parameter_gatherer_enter_scratch_org_expiration_days'
       ),
-      value: defaultExpirationdate
+      value: defaultExpirationdate,
+      validateInput: value => {
```

Comment:
Cool! Thank you for contributing @maaaaarco. Can you add some tests for your validation functions?

Reply:
Sure, will do

Reply:
@brpowell do you think that it would make sense to group these kind of functions in a util file (under the commands folder) ?
In this way we might reuse them for other commands too.

Reply:
I moved those function is a separate file and add the specific tests... I'm a bit lost though when it comes to the _forceOrgCreate.test_...
Because of these
`beforeEach(() => {
      inputBoxSpy = sinon.stub(vscode.window, 'showInputBox');
    });`
I'm not sure how to test that the validation function is being called. Do you have any suggestions/example I can look at?

Reply:
I think the tests you have for the validations themselves are good as is, the vscode api takes care of triggering the validation function

Reply:
Yeah cool, I didn't see that package. I'll do as u suggest ;)